platforms/org.eclipse.mita.platform.arduinouno/ARD_SDK/ARD_LED.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * ARD_LED.h
+ *
+ *  Created on: 24.05.2018
+ *      Author: rherrmann
+ */
+
+#ifndef SRC_GEN_BASE_ARD_LED_H_
+#define SRC_GEN_BASE_ARD_LED_H_
+
+#include "ARD_Retcode.h"
+#include <avr/io.h>
```

Comment:
This include should be in `ARD_LED.c` if you need it. In this header you make no use of anything defined in `io.h`